recipes/mykrobe/post-link.sh

```diff
@@ -28,5 +28,5 @@ fi
 
 tar -vxzf "$TARBALL"
 rm -fr "$PREFIX"/src/mykrobe/data
-mv mykrobe-data "$PREFIX"/src/mykrobe/data
+cp -r mykrobe-data/* "$PREFIX"/lib/python3.6/site-packages/mykrobe/data
```

Comment:
This should be put under `${PREFIX}/share` or something like that, since python 3.6 may or may not be present (in particular, python 3.7 is the default currently).

Reply:
Yes, didn't consider different python versions as when I installed it, it always defaulted to this one. Switching it to share though does not fix the problem and I'm right back to where I started trying to fix this issue: https://github.com/bioconda/bioconda-recipes/issues/13784.

Reply:
@dpryan79 I've implemented a change. It's no longer locked into a specific python version, and the data is stored in share.

Reply:
Great!